Source/Bricks/Label/LabelBrick.swift

```diff
@@ -174,6 +174,7 @@ public class LabelBrickCell: GenericBrickCell, Bricklike {
     @IBOutlet weak public var horizontalRuleLeft: UIView?
     @IBOutlet weak public var horizontalRuleRight: UIView?
     @IBOutlet weak public var imageView: UIImageView?
+    @IBOutlet weak public var chevronImageView: UIImageView?
```

Comment:
ok. Thanks for the explanation
I'd like to see a test for this (see LabelBrickTests)